atlp-rwanda/rca-barfoot-team3-bn
# 184637995 Send Verification Code

#### What does this PR do?
Sending a verification code to user email after registration

#### Description of Task to be completed?
I have used  nodemailer package and javascript function to generate a unique code to send a registration an email to the user containing the code.

#### How should this be manually tested?
Update your env file with the new variables of mail configuration, check the sample from env.example. After that registration your user, check if the message was sent.

#### What are the relevant pivotal tracker/Trello stories?
#184637995  - An Email should be sent to a user upon successful registration. The user should be able to receive a verification link via email upon successful signup
